### PR TITLE
[bootgatthrm] Added BLE sensor location and tuned HRM service advertisement settings

### PR DESCRIPTION
**Pull request by AnotherStranger**
This pull request includes two key changes to enhance the functionality of the BangleJs HRM device and improve compatibility with the OpenTracks app.

 - added BLE sensor location (0x2a38) with value "wrist":
    In this update, I have added the BLE sensor location characteristic with the value "wrist" (0x2a38). This addition provides important information about the placement of the HRM device on the user's body, specifically on the wrist. This information can be utilized by apps and services to accurately identify and interpret the sensor data from the BangleJs HRM device.

- Tuned HRM service advertisement settings for OpenTracks compatibility:
    To ensure that the BangleJs HRM device is discoverable and recognized by the OpenTracks app, I have made specific adjustments to the HRM service advertisement settings. These changes optimize the advertisement parameters and make them compatible with OpenTracks' scanning mechanisms. With these settings in place, OpenTracks will be able to detect and connect to the BangleJs HRM device seamlessly, enabling users to track their heart rate during physical activities.

### CODE REVIEW

File: apps/bootgatthrm/boot.js [hunk 1]
```diff
@@ -23,11 +39,11 @@
...
         0x180D: {
           0x2A37: {
-            value: [
-              0x06, //
-              hrm.bpm
-            ],
+            value: [0x06, hrm.bpm],
             notify: true
+          },
+          0x2A38: {
```
Comment (AnotherStranger) [inline apps/bootgatthrm/boot.js:45]: @gfwilliams Do I need to specify **all** values within a updateServices call, or only changed values?

Comment (gfwilliams) [reply apps/bootgatthrm/boot.js:45]: Only changed values :)

**Comment by gfwilliams** [comment]
Great - sorry my setServices advertising suggestion didn't work though

**Comment by AnotherStranger** [comment]
> Great - sorry my setServices advertising suggestion didn't work though

It worked with FitoTrack! Only OpenTracks filtered the Bangle.

**Pull request merged**